content/docs/pipelines/upgrade.md

```diff
@@ -11,10 +11,10 @@ supports the following capabilities:
 * **Reinstall:** You can delete a cluster and create a new cluster, specifying
   the storage to retrieve the original data in the new cluster.
 
-Note that upgrade isn't currently supported for Kubeflow, check [this issue](https://github.com/kubeflow/kubeflow/issues/3727)
+Note that upgrade isn't currently supported for Kubeflow. Check [this issue](https://github.com/kubeflow/kubeflow/issues/3727)
 for progress.
 
-If you want latest Kubeflow Pipelines in Google Cloud Platform (GCP), we recommend deploying by [Pipelines Standalone Deployment](../standalone-deployment-gcp) instead. It supports upgrade.
+If you want the latest Kubeflow Pipelines in Google Cloud Platform (GCP), we recommend that you use the Kubeflow Pipelines standalone deployment (../standalone-deployment-gcp).
```

Comment:
This needs the square brackets(`[` and `]`)  for the link to work properly (You can see that it renders funny at [this preview](https://deploy-preview-1378--competent-brattain-de2d6d.netlify.com/docs/pipelines/upgrade/))

Additionally, since I'm asking that you change this bit again, I have an additional request: that you change the URL to be relative from website root, instead of using `..` (generally clearer and easier to maintain). So in sum:

```
we recommend that you use the [Kubeflow Pipelines standalone deployment](/docs/pipelines/standalone-deployment-gcp/).
```

Thanks!

Reply:
Fixed.